source/vibe/web/rest.d

```diff
@@ -368,17 +349,15 @@ class RestInterfaceClient(I) : I
 	}
 
 	/// ditto
-	final @property void requestFilter(RequestFilter v) {
+	final @property void requestFilter(RequestFilter v)
+	{
 		m_requestFilter = v;
-		mixin (generateRestInterfaceSubInterfaceRequestFilter!I());
+		foreach (i, SI; Info.SubInterfaceTypes)
+			m_subInterfaces[i].requestFilter = v;
 	}
 
-	//pragma(msg, "subinterfaces:");
-	//pragma(msg, generateRestInterfaceSubInterfaces!(I)());
-	mixin (generateRestInterfaceSubInterfaces!I());
-
 	//pragma(msg, "restinterface:");
-	mixin RestClientMethods!I;
+	mixin(generateRestClientMethods!I());
```

Comment:
See my original PR message. The data available in `RestInterface!T` simply lends itself very well for this and the code got a lot shorter that way. Remember that this is just a thin layer between the `RestInterfaceClient` class and the `executeClientMethod` template function, so the drawbacks of string mixins are unlikely to appear in practice.